app/views/users/_meta.html.erb

```diff
@@ -18,7 +18,7 @@
   <%= auto_discovery_link_tag(:rss, app_url("/feed/#{@user.username}"), title: "#{community_name} RSS Feed") %>
 <% end %>
 
-<% if @user.suspended? %>
+<% if @user.score < 0 @user.suspended? %>
```

Comment:
Typo and syntax error, missing a `||` or a `&`

Reply:
whoops